source/core/src/main/com/csse3200/game/screens/EndMiniGameScreen.java

```diff
@@ -77,6 +107,14 @@ private void setupExitButton() {
             public void clicked(InputEvent event, float x, float y) {
                 // Return to main menu and original screen colour
                 Gdx.gl.glClearColor(248f / 255f, 249f / 255f, 178f / 255f, 1f);
+                switch(getMedal(score)) {
+                    case MiniGameMedals.BRONZE -> display.getEntity().getEvents().trigger("addItem", new EarlyGameLootBox(
```

Comment:
Again, what happens if the players inventory is full? Would it be better to spawn this on the ground near the player?